src/main/views/macro/externalLink.njk

```diff
@@ -0,0 +1,4 @@
+{% macro externalLink(text, href) %}
+  <a href="{{ href }}" target="_blank">{{ text }}</a>
+  <span class="font-xsmall">{{ t('(opens in a new window)') }}</span>
```

Comment:
Thta will bring `(opens in a new window)` to response handoff pages but I guess that is alright. External links should always say they open in new window.